src/lib/journey/callbacks/text-output/text-output.stories.js

```diff
@@ -0,0 +1,43 @@
+import { screen, userEvent } from '@storybook/testing-library';
+import { FRStep, CallbackType } from '@forgerock/javascript-sdk';
+
+import response from './text-output.mock';
+import TextOutput from './text-output.story.svelte';
+
+const step = new FRStep(response);
+
+export default {
+  argTypes: {
+    callback: { control: false },
+    inputName: { control: false },
+  },
+  component: TextOutput,
+  parameters: {
+    layout: 'fullscreen',
+  },
+  title: 'Callbacks/TextOutput',
+};
+
+export const Base = {
+  args: {
+    callback: step.getCallbackOfType(CallbackType.TextOutputCallback),
+    choice: step.getCallbackOfType(CallbackType.ConfirmationCallback),
+  },
+  play : async () => {
+  await new Promise((resolve) => setTimeout(resolve, 100));
+
+  await userEvent.tab();
+
+  const yes = screen.getByText('Yes');
+
+  await userEvent.tab();
+
+  await userEvent.click(yes);
+
+  const no = screen.getByText('No');
+  await userEvent.click(no);
+
+  await userEvent.tab();
+  }
```

Comment:
There's some indentation issue here.

Reply:
Also, since this an interactive component, let's keep the interaction portion, but I'd like to ask it to be consistent with the other implementations. The usual pattern is the following:

1. Base: just the simple, default static rendering of the component. No "play" prop here
2. <Additional variant> (optional): static variants of the component, like primary, secondary, if applicable
3. Error: a static, forced error state of the component
4. Interaction: This is the one where we "play" the interaction and make assertions against the result

What's more, we create the automated "interaction"/"play" version through composition with one of the previous static variants.

```js
const Template = (args) => ({
  Component: Base,
  props: args,
});

export const Interaction = Template.bind({});

Interaction.args = { ...DialogWithTrigger.args };

Interaction.play = async ({ canvasElement }) => {
    // interaction code
}
```

This pattern allows for more robust compositions, and is the one found in most Storybook docs as far as I've seen.

Finally, if we have an interaction, I'd actually like to take the opportunity to assert things against it. We can pull in `expect` from the `jest` library. You can see a simple, yet thorough, example here: https://github.com/cerebrl/forgerock-web-login-framework/blob/main/src/lib/components/compositions/checkbox/animated.stories.js